sixtron_flash/cli.py

```diff
@@ -87,14 +87,19 @@ def main(jlink_device, file_path):
     command_file.write(command_file_content)
     command_file.close()
 
+    if jlink_probe is not None:
```

Comment:
I think it's better to just test like below:
```suggestion
    if jlink_probe:
```
See https://stackoverflow.com/a/3901191 for explanations.